### PR TITLE
Added tcpKeepalive option

### PR DESCRIPTION
**Pull request by dpwspoon**

### CODE REVIEW

File: src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
     public static final String MAXIMUM_BACKLOG_PROPERTY_NAME = "nukleus.tcp.maximum.backlog";
     public static final String WINDOW_THRESHOLD_PROPERTY_NAME = "nukleus.tcp.window.threshold";
     public static final String MAX_CONNECTIONS_NAME = "nukleus.tcp.max.connections";
+    public static final String TCP_KEEP_ALIVE_NAME = "nukleus.tcp.keepalive";
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java]: Rename constant to `KEEPALIVE_NAME` for consistency with naming pattern for other constants.

File: src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java [hunk 2]
```diff
@@ -32,6 +33,8 @@
 
     private static final int MAX_CONNECTIONS_DEFAULT = Integer.MAX_VALUE;
 
+    private static final boolean TCP_KEEP_ALIVE_DEFAULT = false;
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java]: Rename constant to `KEEPALIVE_DEFAULT` for consistency with naming pattern for other constants.

File: src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java [hunk 3]
```diff
@@ -62,4 +65,9 @@ public int maxConnections()
         return getInteger(MAX_CONNECTIONS_NAME, MAX_CONNECTIONS_DEFAULT);
     }
 
+    public boolean tcpKeepalive()
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java]: Rename to `keepalive()`.

File: src/main/java/org/reaktivity/nukleus/tcp/internal/stream/Acceptor.java [hunk 4]
```diff
@@ -57,6 +58,7 @@
 
     private final int backlog;
     private final int maxConnections;
+    private final boolean tcpKeepalive;
```
Comment (jfallows) [inline src/main/java/org/reaktivity/nukleus/tcp/internal/stream/Acceptor.java]: Rename to `keepalive`.

**Comment by dpwspoon** [comment]
@jfallows feedback applied

**Pull request merged**